packages/fxa-settings/src/components/Settings/FlowSetup2faBackupCodeDownload/index.tsx

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect } from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useAccount } from '../../../models';
+import FlowContainer from '../FlowContainer';
+import ProgressBar from '../ProgressBar';
+import { GleanClickEventType2FA } from '../../../lib/types';
+import DataBlock from '../../DataBlock';
+import GleanMetrics from '../../../lib/glean';
+
+type FlowSetup2faBackupCodeDownloadProps = {
+  currentStep?: number;
+  hideBackButton?: boolean;
+  localizedFlowTitle: string;
+  numberOfSteps?: number;
+  onBackButtonClick?: () => void;
+  showProgressBar?: boolean;
+  totpInfo: {
+    recoveryCodes: string[];
+  };
+  onContinue: () => void;
+  reason?: GleanClickEventType2FA;
+};
+
+export const FlowSetup2faBackupCodeDownLoad = ({
+  currentStep,
+  hideBackButton = false,
+  localizedFlowTitle,
+  numberOfSteps,
+  onBackButtonClick,
+  showProgressBar = true,
+  totpInfo,
+  onContinue,
+  reason = GleanClickEventType2FA.setup,
+}: FlowSetup2faBackupCodeDownloadProps) => {
+  const account = useAccount();
+
+  useEffect(() => {
+    GleanMetrics.accountPref.twoStepAuthCodesView({
+      event: { reason: GleanClickEventType2FA.setup },
+    });
+  }, []);
+
+  return (
+    <FlowContainer
+      title={localizedFlowTitle}
+      {...{ hideBackButton, onBackButtonClick }}
+    >
+      {showProgressBar && currentStep && numberOfSteps && (
+        <ProgressBar {...{ currentStep, numberOfSteps }} />
+      )}
+      <FtlMsg id="flow-setup-2fa-backup-code-dl-heading">
+        <h2 className="font-bold text-[24px] my-2">
```

Comment:
Let's use `text-xl` as used in other flow components instead of an arbitrary value.